exchanges.yml

```diff
@@ -432,6 +432,7 @@ components:
           properties:
```

Comment:
Unable to add this comment to the `openId` line, so I will place it here. Can we rename this property to `oid4vp` or `OID4VP`, since that is a more accurate description and could be confusing with respect to the workflow’s `authorization.oauth2` schema?

Reply:
Hmm, well OAuth2 is specifically different from OpenID.

Additionally, the `openId` section covers *both* OID4VP and OID4VCI -- and some steps might require both of these at once. I imagine we'd have to have two separate sections for them, and if anything is shared across the protocols, we might have unnecessary duplication (that usually leads to bugs). So I'm not sure if possible confusion between `authorization.oauth2` and `.openId` is a strong enough reason to rename or split those sections (or to rename the `openId` section to something like `oid4` which may itself lead to other confusions?).